addons/skin.estuary/1080i/Defaults.xml

```diff
@@ -151,9 +151,10 @@
 		<aligny>center</aligny>
 		<texturefocus border="40" colordiffuse="button_focus">buttons/button-fo.png</texturefocus>
 		<texturenofocus border="40">buttons/button-nofo.png</texturenofocus>
+		<orientation>horizontal</orientation>
 		<sliderwidth>150</sliderwidth>
 		<sliderheight>28</sliderheight>
-		<texturesliderbar>buttons/slider-back.png</texturesliderbar>
+		<texturesliderbar border="10">buttons/slider-back.png</texturesliderbar>
```

Comment:
indeed. you'll need a new set of textures for vertical settings sliders.
it's the same for slider and scrollbar controls, skins ship with separate textures for horizontal and vertical ones.